src/_P103_Atlas_EZO_pH_ORP_EC.ino

```diff
@@ -0,0 +1,641 @@
+// ########################################################################
+// ################## Plugin 103 : Atlas Scientific EZO pH ORP EC sensors #
+// ########################################################################
+
+// datasheet at https://atlas-scientific.com/files/pH_EZO_Datasheet.pdf
+// datasheet at https://atlas-scientific.com/files/ORP_EZO_Datasheet.pdf
+// datasheet at https://atlas-scientific.com/files/EC_EZO_Datasheet.pdf
+// works only in i2c mode
+
+# include "src/Helpers/Rules_calculate.h"
+
+# define PLUGIN_103
+# define PLUGIN_ID_103           103
+# define PLUGIN_NAME_103         "Environment - Atlas EZO pH ORP EC"
+# define PLUGIN_VALUENAME1_103   "SensorData"
+# define PLUGIN_VALUENAME2_103   "Voltage"
+# define UNKNOWN                 0
+# define PH                      1
+# define ORP                     2
+# define EC                      3
+
+# define FIXED_TEMP_VALUE        20   // Temperature correction for pH and EC sensor if no temperature is given from calculation
+
+boolean Plugin_103(byte function, struct EventStruct *event, String& string)
+{
+  boolean success = false;
+
+  byte board_type;
+  byte I2Cchoice;
+
+  switch (function)
+  {
+    case PLUGIN_DEVICE_ADD:
+    {
+      Device[++deviceCount].Number           = PLUGIN_ID_103;
+      Device[deviceCount].Type               = DEVICE_TYPE_I2C;
+      Device[deviceCount].VType              = Sensor_VType::SENSOR_TYPE_DUAL;
+      Device[deviceCount].Ports              = 0;
+      Device[deviceCount].PullUpOption       = false;
+      Device[deviceCount].InverseLogicOption = false;
+      Device[deviceCount].FormulaOption      = true;
+      Device[deviceCount].ValueCount         = 2;
+      Device[deviceCount].SendDataOption     = true;
+      Device[deviceCount].TimerOption        = true;
+      Device[deviceCount].GlobalSyncOption   = true;
+      break;
+    }
+
+    case PLUGIN_GET_DEVICENAME:
+    {
+      string = F(PLUGIN_NAME_103);
+      break;
+    }
+
+    case PLUGIN_GET_DEVICEVALUENAMES:
+    {
+      strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[0], PSTR(PLUGIN_VALUENAME1_103));
+      strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[1], PSTR(PLUGIN_VALUENAME2_103));
+      break;
+    }
+
+    case PLUGIN_WEBFORM_LOAD:
+    {
+      I2Cchoice  = PCONFIG(1);
+      addHtml(F("<TR><TD>Default I2C Address:</TD><TD>pH: 0x63, ORP: 0x62, EC: 0x64. The plugin is able to detect the type of device automatically.</TD></TR>"));
+      # define _P103_ATLASEZO_I2C_NB_OPTIONS 6
+      int optionValues[_P103_ATLASEZO_I2C_NB_OPTIONS] = { 0x62, 0x63, 0x64, 0x65, 0x66, 0x67 };
+      addFormSelectorI2C(F("plugin_103_i2c"), _P103_ATLASEZO_I2C_NB_OPTIONS, optionValues, I2Cchoice);
+
+      addFormSubHeader(F("General"));
+
+      char sensordata[32];
+      bool info;
+      info = _P103_send_I2C_command(I2Cchoice, F("i"), sensordata);
+
+      if (info) {
+        String boardInfo(sensordata);
+
+        addHtml(F("<TR><TD>Board type : </TD><TD>"));
```

Comment:
Why not using `addRowLabel` here?
This way we can have a more uniform look and feel of the UI.

Reply:
yes. I will fix this.

Reply:
fixed